Main.py

```diff
@@ -9,7 +9,16 @@
 from ExtractMainContent import ExtractMainContent
 import sys
 import codecs
-
+import signal
+"""
+this function handles signal SIGINT generated when user presses ctrl-c

```

Comment:
In Python, there is a general guideline that doc string of the function should be inside the function definition part.
for e.g  def signal ():
               """this function does these things"""